status-im/status-react
# Implement display of missing key pairs in wallet settings

[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

fixes #20033 
fixes #19733

### Summary

* This PR attempts to implement the logic for displaying the missing key-pairs available from a user's account, but not available on the user's device.
* This PR attempts to implement the Quo components for missing-keypairs list and list items.
* This PR attempts to resolve some small bugs with the "Key pairs and accounts" settings screen
* This PR attempts to resolve an issue with Quo standard-title component

#### Platforms

- Android
- iOS

#### Areas that maybe impacted


##### Functional

- Mobile wallet settings
- Key pairs and accounts settings

### Steps to test

To test these changes there are some requirements, the user account for testing will need with two devices, and both of those devices must be synced to the user account.

Once the account and devices are setup to sync between the devices:
* Open the Status app (mobile or desktop) on one device and create a new account with a key-pair.
  * Choose to create a new account from the wallet home screen, then select to create a new key-pair by generating a recovery phrase key-pair, and then finish creating the account with that new key-pair.
  * Because the devices are synced, the second device should be updated with the knowledge of the new account and the key-pair.
* Navigate to the mobile wallet setting and open the "Key pairs and accounts" settings.
* At the top of screen there should be a section for missing key-pairs, and that section should display the current missing key-pair and the icon emojis for the related accounts.



### After changes screen capture

https://github.com/status-im/status-mobile/assets/2845768/7a728135-4747-4c9b-af01-47d1ad92fd12

status: ready
